### PR TITLE
Bring back heatmap layer to give users more context

### PR DESCRIPTION
**Pull request by vnugent**
(Re)introduce the heatmap layer to give users more context of what there is in the area regardless of user's filters.  The heatmap layer can also be used for other pages.

Screen Shot 2022-04-23 at 3 03 16 PM

---

### close up view (zoomed in)
- turn off heatmap
- show areas not in user preferences in smaller font and marker

Screen Shot 2022-04-23 at 3 05 35 PM

### CODE REVIEW

File: src/components/maps/MarkerLayer.tsx [hunk 1]
```diff
@@ -1,28 +1,59 @@
...
+    'text-size': ['interpolate', ['linear'], ['zoom'], 8, 12, 14, 10],
+    'icon-allow-overlap': false
+  },
+  paint: {
+    'icon-color': '#ffffff',
+    'text-halo-blur': 4,
+    'text-halo-width': 2,
+    'text-color': '#a3a3a3',
     'text-halo-color': '#334455'
+    // 'text-opacity':
```
Comment (MichaelDimmitt) [inline src/components/maps/MarkerLayer.tsx]: css variables could be used
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

to centralize all of your colors.

**Comment by MichaelDimmitt** [comment]
Is there a way to test that it is operating correctly?

Though maybe we do not need to test the inner workings of 3rd party libraries.
I could go either way.

File: src/components/maps/MarkerLayer.tsx (hunk 1 above)
Comment (vnugent) [reply src/components/maps/MarkerLayer.tsx]: Thanks.  I opened https://github.com/OpenBeta/open-tacos/issues/287

**Comment by vnugent** [comment]
@MichaelDimmitt we should be at least testing the global store, but we're not at the moment.  For custom map components I guess we can mock Mapbox/webgl coomponents and use jest snapshots for the rest?

**Pull request merged**